conda_build/os_utils/liefldd.py

```diff
@@ -890,8 +890,10 @@ def get_symbols(file, defined=True, undefined=True, notexported=False, arch='nat
         except:
             pass
     res = []
+    imported_function_names = set()
     if len(binary.exported_functions):
         syms = binary.exported_functions
+        imported_function_names = {s.name for s in binary.imported_functions}
```

Comment:
I don't really know if this is the correct thing to do. Just made sense given the names of `lief.Binary`'s members.